### PR TITLE
Updates following #347

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Updates after https://github.com/EGI-Federation/fedcloud-catchall-operations/pull/347/

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by sebastian-luna-valero** [comment]
Thanks @gwarf 

I will wait for @enolfc 's approval next week.

File: deploy/cloud-init.yaml
Comment (enolfc) [inline deploy/cloud-init.yaml:14]: This was added to have a recent ansible (xenial one that we were getting was too old), so I think this should stay

Comment (sebastian-luna-valero) [reply deploy/cloud-init.yaml:14]: But current image is 22.04, right?
https://github.com/EGI-Federation/fedcloud-catchall-operations/blob/58c38e2db0507677b6e8f1538921105dc984bb99/deploy/NCG-INGRID-PT.tfvars#L7

Which is coming with a recent ansible version?
https://packages.ubuntu.com/jammy/ansible

Comment (enolfc) [reply deploy/cloud-init.yaml:14]: ok, let's merge

**Pull request merged**
